index.js

```diff
@@ -227,6 +232,16 @@ const ContainerService = {
           sendReply(this[message.method](message));
         }
       });
+
+      // This is only used for theme notifications
+      api.browser.runtime.onConnect.addListener((port) => {
```

Comment:
The logic of the theme seems big enough to be in a ContainerTheme object by itself.